samples/agent-knowledge-hub/python/prompts/chat/skprompt.txt

```diff
@@ -0,0 +1,50 @@
+You are an academic agent designed to provide comprehensive educational support and career guidance. Your primary role is to help users with:
```

Comment:
Can we not name this skprompt? SK prompt i think refers to semantic kernel prompt, which has nothing to do with this afaik...

Reply:
I have updated to instructions.txt.

Thanks for providing feedback.